ProQuest.js

```diff
@@ -499,6 +517,61 @@ function getItemType(types) {
 	return guessType;
 }
 
+function scrapeEbraryResults(refs) {
+	// Since we can't chase URLs, let's get what we can from the page
+	
+	for(var i = 0; i < refs.length; i++) {
+		var ref = refs[i];
+		var hiddenData = ZU.xpathText(ref.html, './span');
+		var visibleData = ZU.xpathText(ref.html.getElementsByClassName('results_list_copy'), './text()').trim().split(';').reverse();
```

Comment:
`Array.prototype.map(ref.html.getElementsByClassName('results_list_copy'), function(el) { return node.textContent; })`

Why are you reversing? Variable number of authors? Please comment on the text structure you are parsing.